ch1.adoc

```diff
@@ -0,0 +1,521 @@
+= Introduction
+
+This tutorial will introduce you to write your own programming
+language using Cranelift as a backend.
```

Comment:
Grammar: change "write" to "writing". Or perhaps say "will show you how to write your own programming language".